README.MD

```diff
@@ -29,6 +29,7 @@ It's always been sad to see a service shutdown, or hardware stop renew, here is
 - [Wearables Table](https://quantifyme.io/) - Nice crowd-sourced table of wearables.
 - [Wolfram Connected Devices Project](http://devices.wolfram.com/) - IoT and Quantified Self device list.
 - [Quantified Self Instutite](http://www.qsinstitute.org/) - Quantified Self research and innovation projects.
+- [Biomindmap](https://biomindmap.com/) - Very nice database of interventions with a network viz option. 
```

Comment:
"Very nice" seems like an opinion. Can we remove that?

Reply:
Very well.

Reply:
I did but idk github review process that well.